proxy/http/HttpTransact.cc

```diff
@@ -178,12 +178,22 @@ inline static void
 findParent(HttpTransact::State *s)
 {
   url_mapping *mp = s->url_map.getMapping();
-
-  if (mp && mp->strategy) {
-    return mp->strategy->findNextHop(reinterpret_cast<TSHttpTxn>(s->state_machine));
+  if (s->response_action.handled) {
+    s->parent_result.hostname = s->response_action.action.hostname;
+    s->parent_result.port     = s->response_action.action.port;
+    s->parent_result.retry    = s->response_action.action.retry;
```

Comment:
I don't understand this value. Shouldn't the core determine / know if it's a retry? The plugin shouldn't be the authority on that.

Reply:
Looking at the HostDB restructure and other discussions, the core will need to track the number of retries per upstream and per transaction.